src/reducerFunctions/filterReducer/reducerUtilities/filterSearch.jsx

```diff
@@ -0,0 +1,10 @@
+
+export const filterBySearch = (productData, searchItem) => {
+    if (searchItem) {
+        return productData.filter(productItem =>
+            productItem.name.toLowerCase().includes(searchItem.toLowerCase()));
+    }
+    return productData;
+
+}
```

Comment:
liked that you converted the query into lowercase

Reply:
thank you